### ISSUE TITLE
Finetune on kitti 2015

### ISSUE THREAD

**Issue by Archerwz** [mileyan/AnyNet#13]
Hi, I wonder how you finetune on Kitti 2015 without overfitting. Because, when I tried to finetune on Kitti 2015 with random crop, the validation loss start to increase at epoch 20, further training can only make the metrics on the validation set worse. 
For finetune I split the 200 images to 160 as train set and 40 as val set.

**Comment by mileyan** [comment]
Hi, you can try to use Adam optimizer and use a smaller learning rate. Could you provide some training code, so I can help you do an analysis?

**Comment by Archerwz** [comment]
So I change your network a little bit, instead of one disparity map, I form two cost volume and concat them together for later output disp map for both right and left image.
I pretrain on the kitti raw dataset with official annotated depth map (contain groundtruth for both left and right) for 10 epochs which I convert them to disparity with focal length and baseline. I later finetune on kitti 2015, since it only has ground truth for left images, I warp the predicted right disparity to left and compute its lose with respect to left groundtruth.

**Comment by mileyan** [comment]
Hi, how you warp the predicted right disparity to left? I suggest you first project the left image ground truth to 3d coordinate and then use the point cloud to generate the right image ground truth.
